kentik_api_library/kentik_api/api_resources/custom_dimensions_api.py

```diff
@@ -97,9 +103,12 @@ def get_all(self) -> List[CustomDimension]:
         return custom_dimensions_payload.GetAllResponse.from_json(response.text).to_custom_dimensions()
 
     def create(self, custom_dimension: CustomDimension) -> CustomDimension:
-        assert custom_dimension.name is not None
-        assert custom_dimension.display_name is not None
-        assert custom_dimension.type is not None
+        if custom_dimension.name is None:
+            raise IncompleteObjectError("Custom Dimensions", "custom dimension name value is required")
```

Comment:
```suggestion
            raise IncompleteObjectError("Custom Dimensions", "name is required")
```
In other places too.

Reply:
Done